docs/commands/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _commands-index:
+
+
+index
+=========
+
+Index a list of haplotypes.
+
+The ``index`` command takes as input a list of haplotypes (specified as a :doc:`.hap file </formats/haplotypes>`) and outputs the list as a .gz and a .tbi file.
```

Comment:
It might also be good to mention that the command also automatically sorts the file here.